Added preview of As-Is and To-Be workpieces in the WorkpieceList

MikeDiBuduo opened the pull request:
Added functionality so when the preview button next to the workpiece in the workpiece list is clicked, the preview pane pops up with the workpiece.

MikeDiBuduo committed:
commiting so I can pull down changes from class. Started working on the preview button for the work piece

MikeDiBuduo committed:
merged my code starting the preview with the code from class

MikeDiBuduo committed:
Added the ability to preview the workpiece when you click on the icon next to the workpiece in the workpiece list.

Sam-son commented:
Content looks good, couple notes-  
Why did you change proptypes of WorkingstepList?  
Why did you comment out the DatumList and WorkpieceList in ToleranceProperties?

MikeDiBuduo commented:
The proptypes change was an accident. I must have done it when trying to change the proptypes of the WorkpieceList. I commented out the other things because that's what we did in class after you made the commit.

MikeDiBuduo committed:
Removed unneccesary changes

Removed accidental change of proptypes of WorkingstepList and uncommented the DatumList and WorkpieceList in the TolerancePropeties
